viewer/db.js

```diff
@@ -118,6 +121,18 @@ exports.initialize = async (info, cb) => {
     setInterval(() => { exports.getAliasesCache('sessions2-*'); }, 2 * 60 * 1000);
   }
 
+  // if there's a user's db, sync shortcuts from user's to local db so they can be
+  // used for sessions search (can't use remote db for searching via shortcuts)
+  internals.localShortcutsIndex = fixIndex('lookups');
+  if (internals.info.usersHost) {
```

Comment:
Only do this stuff if its a cron query viewer node AND usersElasticserach is set